pages/agent/v3/securing.md

```diff
@@ -41,6 +41,11 @@ Command line evaluation can be disabled by setting [`no-command-eval`](/docs/age
 >🚧 Custom hooks
 > If you have a custom <code>command</code> hook, using <code>no-command-eval</code> will have no effect on your command execution. See <a href="#allowing-a-list-of-plugins">Allowing a list of plugins</a> and <a href="#customizing-the-bootstrap">Custom bootstrap scripts</a> for examples of how to completely lock down your agent from arbitrary code execution.
 
+<!-- comment to ensure we create two separate callout blocks -->
+
+>🚧 Environment variables
+> Using <code>no-command-eval</code> only prevents command evaluation by the agent itself. Other programs that run during the job could be influenced into executing arbitrary commands via environment variables (for example, `BASH_ENV` or `GIT_SSH_COMMAND`). See <a href="#strict-checks-using-a-pre-bootstrap-hook">Strict checks using a pre-bootstrap hook</a> and <a href="/docs/agent/v3/cli-start#enable-environment-variable-allowlist"><code>enable-environment-variable-allowlist</code></a> for possible approaches to filtering environment variables.
```

Comment:
Following on from above ...

```suggestion
<!-- Markdown comment divider create two separate callout blocks -->

> 🚧 Environment variables
> Using `no-command-eval` only prevents command evaluation by the agent itself. Other programs that run during the job could be influenced into executing arbitrary commands via environment variables (for example, `BASH_ENV` or `GIT_SSH_COMMAND`). See [Strict checks using a pre-bootstrap hook](#strict-checks-using-a-pre-bootstrap-hook) and [`enable-environment-variable-allowlist`](/docs/agent/v3/cli-start#enable-environment-variable-allowlist) for possible approaches to filtering environment variables.
```

When you say "Other programs", what exactly is meant by 'programs'? What might be an example of such a program?